src/app/content/highlights/components/DisplayNote.tsx

```diff
@@ -39,8 +39,8 @@ interface Props {
 }
 
 // tslint:disable-next-line:variable-name
-const DisplayNote = React.forwardRef<HTMLElement, Props>((
-  {note, isFocused, onBlur, onEdit, onRemove, onHeightChange, className}: Props,
+const DisplayNote = React.forwardRef<HTMLElement, DisplayNoteProps>((
+  {note, isFocused, onBlur, onEdit, onRemove, onHeightChange, className}: DisplayNoteProps,
```

Comment:
Does it need the `DisplayNoteProps` in destructuring if you pass it to the generic?